lib/src/room.dart

```diff
@@ -242,6 +248,8 @@ class KurentoRoom extends ChangeNotifier implements Room {
 
   String get _serviceRequestPresenceTopic => '${_env.name}/user/${_serviceRequest.userId}/service-request/presence';
 
+  String get _triggerTopic => '${_env.name}/si/ts/${_serviceRequest.userId}';
```

Comment:
I don't like the name `_triggerTopic`, but I see that Platform refers to these things as "service triggers". 🤦🏾

Reply:
I did attempt to find a better name... this channel is used to trigger things. The other name I found (again inspired from legacy) is 'ToSmart' hehehe! Not amazing either. I feel we will need to change some of those topics, merge some of them as it seems like the information about the GPS activated offers should come through the same topic as the flashlight. That trigger topic has other things like the 'flashlight' in there as well.

Reply:
I've renamed it to `_callEventsTopic`